k8s-daemonset/k8s-ingress/linkerd.yml

```diff
@@ -0,0 +1,127 @@
+# runs linkerd in a daemonset, in linker-to-linker mode
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: l5d-config
+data:
+  config.yaml: |-
+    admin:
+      port: 9990
+
+    namers:
+    - kind: io.l5d.k8s
+      experimental: true
+      host: localhost
+      port: 8001
+
+    routers:
+    - protocol: http
+      label: ingress
+      baseDtab: |
+        /world/hello/www => /#/io.l5d.k8s/default/http/hello ;
+        /world/hello/api => /#/io.l5d.k8s/default/http/api ;
+        /host            => /$/io.buoyant.http.domainToPath ;
+        /http/*/*        => /host ;
+      servers:
+      - port: 80
+        ip: 0.0.0.0
```

Comment:
Instead of serving this router on port 80, I think it would be more conventional to have it serve on a higher port (4142?), and then map that port to port 80 in the l5d service object.